source/_posts/19-09-05组会.md

```diff
@@ -0,0 +1,15 @@
+---
+title: RelGAN: Relational generative adversarial networks for text generation
```

Comment:
I cant open this website page [190905](https://ecnuprml.github.io/2019/09/19-09-05%E7%BB%84%E4%BC%9A/) maybe the markdown file name needs to be English??